cardano-node/src/Cardano/Node/Types.hs

```diff
@@ -214,9 +214,11 @@ instance FromJSON NodeConfiguration where
       parseHardForkProtocol v = do
         npcTestShelleyHardForkAtEpoch   <- v .:? "TestShelleyHardForkAtEpoch"
         npcTestShelleyHardForkAtVersion <- v .:? "TestShelleyHardForkAtVersion"
+        npcShelleyHardForkNotBeforeEpoch <- v .:? "ShelleyHardForkNotBeforeEpoch"
```

Comment:
Should the default configuration file for mainnet define `ShelleyHardForkNotBeforeEpoch` to be 190, or in fact 208?

Reply:
We'll put it into the actual config file, so it's explicit.

Reply:
That's what I meant.